### ISSUE TITLE
Un-shitting food

### ISSUE THREAD

**Issue by dontdothis** [Apollo-Community/ApolloStation#1367]
Easy enough fix.  Change nutriment's feed amount to 4x what it currently is (60 instead of 15).  That way, people will fill up at a more reasonable pace instead of having to eat an entire vending machine of donuts.

 ApolloStation/code/modules/reagents/Chemistry-Reagents.dm
Line 2090

**Comment by Draco16** [comment]
The vending machine food is intended to be snacks only not legit food. Currently vending machine food will stop giving nutrients once you get close to green fullness.

**Comment by NovusLuna** [comment]
He does have some degree of a point. If you haven't eaten in four hours, you shouldn't suddenly have to eat three or four whole pizza's.

**Comment by rosextango** [comment]
But you should have to eat around every three to five hours. Assuming everyone's running around and each floor tile is approx. 5 ft (seeing as how that the basis for d&d, but who knows). Sure, you shouldn't have to eat three pizzas, but mayybbe a pizza.

**Comment by Draco16** [comment]
You also shouldn't be able to build an entire engine in 10 minutes or build an entire airtight wall/window in about 3 seconds. Time is not relative to much of anything on SS13.  Large amounts of food is required because being able to sate your entire hunger using nothing but 2 boxes of raisins from the vending machine would make the entire kitchen and staff obsolete.  

Also, for your pizza example, pizzas are generally considered to be personal pizzas when compared to other cook-able foods in SS13.  As a result, eating 2-3 when you are nearly starving to death is perfectly reasonable.  

On top of any of these facts, it does also stand that chefs (who actually do their jobs) and want to be able to server more than one thing to someone; not simply hand someone a single burger and have then be good for the entire shift.  Kinda hard to RP when no one needs to stick around. 

TLDR: Food is balanced.

**Comment by TheBrightDawn** [comment]
Oh Imma' 'bout to mess this shit up with some new foods and shit.

**Comment by Andrew-Fall** [comment]
Can this be closed then?
